### ISSUE TITLE
instructions don't work in osx

### ISSUE THREAD

**Issue by saturnism** [GoogleCloudPlatform/gcsfuse#107]
````
$ brew tap homebrew/fuse
Warning: Already tapped!

$ brew install gcsfuse
Error: undefined method `desc' for Formulary::Formulae::Gcsfuse:Class
Please report this bug:
    http://git.io/brew-troubleshooting
/usr/local/Library/Taps/homebrew/homebrew-fuse/Formula/gcsfuse.rb:4:in `<class:Gcsfuse>'
/usr/local/Library/Taps/homebrew/homebrew-fuse/Formula/gcsfuse.rb:3:in `load_file
````

````
$ go get -u github.com/googlecloudplatform/gcsfuse
# github.com/jacobsa/gcloud/httputil
Applications/go/src/github.com/jacobsa/gcloud/httputil/request.go:59: unknown http.Request field 'Cancel' in struct literal
# github.com/jacobsa/fuse/internal/buffer
Applications/go/src/github.com/jacobsa/fuse/internal/buffer/runtime.go:22: missing function body
Applications/go/src/github.com/jacobsa/fuse/internal/buffer/runtime.go:27: missing function body
`````

**Comment by jacobsa** [comment]
Just 30 seconds ago I reworked the OS X instructions to no longer require homebrew (whose version of gcsfuse was ancient anyway). Please try [the latest version](https://github.com/GoogleCloudPlatform/gcsfuse/blob/7d44c21ce508c65c9c74207d6cbd8c0a01daaf3d/docs/installing.md) and let me know whether it works.

**Comment by saturnism** [comment]
A couple of issues:

the tar -C command actually modifies the /usr/local/bin user/group and permission! The owner:group became 1000:1000, and permission was rwx------

secondly, running gcsfuse command in osx resulted in errors:
````
panic: user: Current not implemented on darwin/amd64

goroutine 1 [running]:
github.com/googlecloudplatform/gcsfuse/internal/perms.MyUserAndGroup(0x0, 0x882099f028, 0x8208ca5f0)
	/tmp/build_release_gopath131858037/src/github.com/googlecloudplatform/gcsfuse/internal/perms/perms.go:29 +0x9a
main.mount(0x88209a3710, 0x820800330, 0x7fff5fbff498, 0x19, 0x7fff5fbff4b2, 0x4, 0x820844bd0, 0x88209a3f38, 0x8208d40c0, 0x0, ...)
	/tmp/build_release_gopath131858037/src/github.com/googlecloudplatform/gcsfuse/mount.go:58 +0x1f5
main.main.func1(0x8208cc000)
	/tmp/build_release_gopath131858037/src/github.com/googlecloudplatform/gcsfuse/main.go:259 +0x556
github.com/googlecloudplatform/gcsfuse/vendor/github.com/codegangsta/cli.(*App).Run(0x82087a500, 0x8207f8510, 0x3, 0x3, 0x0, 0x0)
	/tmp/build_release_gopath131858037/src/github.com/googlecloudplatform/gcsfuse/vendor/github.com/codegangsta/cli/app.go:164 +0xc81
main.main()
	/tmp/build_release_gopath131858037/src/github.com/googlecloudplatform/gcsfuse/main.go:280 +0xf9

goroutine 5 [syscall]:
os/signal.loop()
	/home/jacobsa/clients/go/src/os/signal/signal_unix.go:22 +0x18
created by os/signal.init.1
	/home/jacobsa/clients/go/src/os/signal/signal_unix.go:28 +0x37

goroutine 6 [chan receive]:
main.handleCPUProfileSignals()
	/tmp/build_release_gopath131858037/src/github.com/googlecloudplatform/gcsfuse/main.go:90 +0x1c3
created by main.main
	/tmp/build_release_gopath131858037/src/github.com/googlecloudplatform/gcsfuse/main.go:220 +0x44

goroutine 7 [chan receive]:
main.handleMemoryProfileSignals()
	/tmp/build_release_gopath131858037/src/github.com/googlecloudplatform/gcsfuse/main.go:138 +0x193
created by main.main
	/tmp/build_release_gopath131858037/src/github.com/googlecloudplatform/gcsfuse/main.go:221 +0x5c

goroutine 17 [select, locked to thread]:
runtime.gopark(0x5b5dd0, 0x820810728, 0x4e75f8, 0x6, 0x35418, 0x2)
	/home/jacobsa/clients/go/src/runtime/proc.go:185 +0x163
runtime.selectgoImpl(0x820810728, 0x0, 0x18)
	/home/jacobsa/clients/go/src/runtime/select.go:392 +0xa64
runtime.selectgo(0x820810728)
	/home/jacobsa/clients/go/src/runtime/select.go:212 +0x12
runtime.ensureSigM.func1()
	/home/jacobsa/clients/go/src/runtime/signal1_unix.go:227 +0x31d
runtime.goexit()
	/home/jacobsa/clients/go/src/runtime/asm_amd64.s:1696 +0x1
````

**Comment by jacobsa** [comment]
Yikes, sorry about the permissions issue. Will fix asap.

Regarding the panic: that's very odd. What version of OS X are you running? Is Go installed, and if so can you paste the output of `go version` and `go env`?

**Comment by jacobsa** [comment]
Ah, I can reproduce the issue. And I think I understand what's going on: cross-compiling from Linux causes the `cgo` build tag to be unset, which causes `os/user` to not be supported. Solution: don't compile from Linux.

Sorry for the trouble, will fix.

**Comment by jacobsa** [comment]
@saturnism: I believe both issues are fixed in [v0.8.1](https://github.com/GoogleCloudPlatform/gcsfuse/releases/tag/v0.8.1). Could you please try it out and confirm?

**Comment by saturnism** [comment]
0.8.1 didn't work but 0.9.1 seems to be working :) thanks!
